src/main/java/com/lginterview/dto/CourseStatus.java

```diff
@@ -0,0 +1,83 @@
+package com.lginterview.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.ZonedDateTime;
+
+public class CourseStatus {
+
+    private final ZonedDateTime startedDate;
+    private final int totalSessionTime;
+    private final ZonedDateTime endDate;
```

Comment:
[MS][Trivial] Because endDate may be null, it should be actually Optional<ZonedDateTime>, because now every client of CourseStatus class have to remember to always check if endDate != null.

Reply:
I Agree